### PR TITLE
Adding check_mk scripts (TASK-686)

### PR DESCRIPTION
**Pull request by kkellerlbl**
Also moving aweadmin script to its own dir, and creating placeholder for the docker-compose config files for AWE clients.

### CODE REVIEW

**Comment by kkellerlbl** [comment]
I don't think there's any sensitive information in this PR but it'd be good for someone else to look and make sure.

File: check_mk/check_galera.sh
Comment (sychan) [inline check_mk/check_galera.sh]: Is this sensitive?

**Review by sychan — CHANGES_REQUESTED** [review]
Do the MySQL creds need to be stripped out?

File: check_mk/check_galera.sh
Comment (kkellerlbl) [reply check_mk/check_galera.sh]: YES, oops.  The mysql servers aren't accessible off network but it's still bad to have it in there.

Comment (kkellerlbl) [reply check_mk/check_galera.sh]: Those credentials also only work from certain hosts.  But if there's an easy to strip those creds from my git history before merging that'd be good.

Comment (kkellerlbl) [reply check_mk/check_galera.sh]: Please don't merge yet, I'm trying to sanitize my repo.

Comment (kkellerlbl) [reply check_mk/check_galera.sh]: I think my repo is sanitized now.  Unfortunately the credentials still appear in the text of the PR.  I'm not super paranoid about it but if there's a way to get rid of it that'd be good too.

Comment (kkellerlbl) [reply check_mk/check_galera.sh]: Our mysql creds are now changed, so it's probably safe to merge this PR if there are no other problems.

**Pull request merged**
